### PR TITLE
Downgrade 'invalid content address' error to trace log

### PR DESCRIPTION
**Pull request by janos**

### CODE REVIEW

**Comment by nonsense** [comment]
So I assume this is an expected condition for MRU?

**Review by nonsense — APPROVED** [review]

**Comment by janos** [comment]
@nonsense @zelig, this PR is updated, could you check it again, please?

**Review by nonsense — APPROVED** [review]

File: swarm/storage/localstore.go [hunk 1]
```diff
@@ -97,13 +97,20 @@ func NewTestLocalStoreForAddr(params *LocalStoreParams) (*LocalStore, error) {
 // After the LDBStore.Put, it is ensured that the MemStore
 // contains the chunk with the same data, but nil ReqC channel.
 func (ls *LocalStore) Put(chunk *Chunk) {
+	if l := len(chunk.SData); l < 9 {
+		log.Debug("incomplete chunk data", "addr", chunk.Addr, "length", l)
+		chunk.SetErrored(ErrChunkInvalid)
+		chunk.markAsStored()
```
Comment (nolash) [inline swarm/storage/localstore.go:103]: this method should change name!

**Review by nolash — APPROVED** [review]

**Comment by nolash** [comment]
> So I assume this is an expected condition for MRU?

@nonsense yes

File: swarm/storage/localstore.go (hunk 1 above)
Comment (janos) [reply swarm/storage/localstore.go:103]: Yes, that will change in chunk refactor.

**Comment by janos** [comment]
Thank you for reviews.

**Pull request merged**